code/modules/mob/living/simple_animal/hostile/space_dragon.dm

```diff
@@ -1,9 +1,3 @@
-/// The carp rift is currently charging.
-#define CHARGE_ONGOING 0
-/// The carp rift is currently charging and has output a final warning.
-#define CHARGE_FINALWARNING 1
-/// The carp rift is now fully charged.
-#define CHARGE_COMPLETED 2
 /// The darkness threshold for space dragon when choosing a color
 #define DARKNESS_THRESHOLD 50
```

Comment:
Not really related, but I do wish this define wasn't limited to space dragon, it should have more uses in other places as well I bet.

Reply:
There's a version of this define used for lizards, however, it's broken I'd assume because SD used to share its value with lizards, but randomly after a time pitch black SDs starting showing up and I had to change it. I never play lizard, but I'm guessing black lizards would not be currently be blocked due to that? Unless someone changed it already, of course.